server/src/main/java/org/opensearch/identity/tokens/TokenType.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.identity.tokens;
+
+/**
+ * Enum containing supported token types
+ */
+public enum TokenType {
+    BASIC,
+    BEARER,
+    NONE
```

Comment:
None would mean no auth token. If there is no implementation of a ScheduledJobIdentityManager than it would send blank in place of any auth token to the job runner.